x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/guided_onboarding/translations.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const INSTALL_PREBUILT_RULES_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.guidedOnboarding.installPrebuiltRules.title',
+  {
+    defaultMessage: 'Load the Elastic prebuilt rules',
+  }
+);
+
+export const INSTALL_PREBUILT_RULES_CONTENT = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.guidedOnboarding.installPrebuiltRules.content',
+  {
+    defaultMessage: 'To get started you need to load the Elastic prebuilt rules.',
```

Comment:
```suggestion
    defaultMessage: 'Prebuilt rules are the fastest way to query your data for security events.',
```
This adds some context about _why_ they should use our prebuilt rules. The title already tells them to load them, so we don't need to repeat that.